build/DockerfileWithOpentracing

```diff
@@ -0,0 +1,83 @@
+ARG NGINX_VERSION=1.17.3
```

Comment:
Opentracing was not included in the 1.5 release. It also won't work with just the Dockerfile included.

Please remove, unless I am mistaken.